### DISCUSSION TITLE
Recent GitHub Projects Update - Request for UI Customization

### DISCUSSION THREAD

**Discussion by sht** [community/community#107529] [Projects and Issues]
## Annoying GitHub Project activity logs

Hello GitHub Community,

I want to share some feedback regarding the recent changes made to GitHub Projects, specifically the integration of its UI with the Issues feature.

The previous UI of GitHub Projects was something I found very efficient and user-friendly. However, with the recent update aligning it with the Issues UI, I've observed a significant increase in the activity logs displayed after each comment. These logs include information such as "self-assignment," milestone additions, and other activities.

![Screenshot 2024-02-11 at 12 58 18 PM](https://github.com/community/community/assets/13151141/b22a5297-3b73-4bc6-8b1a-724fcd921f8f)

In the earlier version, the simplicity of focusing solely on comments allowed for a cleaner and more streamlined user experience. For users like myself, who prioritize concentration on the ongoing discussion, the current UI can be overwhelming with the additional activity logs.

I suggest introducing customization options for the GitHub Projects UI. Specifically, I would suggest an option to toggle the visibility of these activity logs. This way, users could choose whether to see detailed activity updates or prefer a more simplified view with only comments.

Adding customization options like these would significantly enhance the user experience, allowing users to tailor the interface to their preferences. This feature could be handy for those who use GitHub Projects predominantly for project management and wish to maintain the simplicity and clarity the platform initially offered.

By incorporating such customization options, GitHub can continue to cater to the diverse needs of its user base and ensure a more enjoyable and efficient experience for everyone.

I welcome your thoughts on this matter and hope for a constructive discussion within the community.

Annoying part:
![Screenshot 2024-02-11 at 12 58 25 PM](https://github.com/community/community/assets/13151141/c2ab531b-83a3-4df7-8c32-3a93c6fcdd53)

Please let them be in "Issues", if someone want to see, they can see in issues, I wouldn't like to have them in "Projects"

**Comment by Akash1134** [comment]
Welcome to the GitHub Community, @sht , we're happy you're here! Your discussion belongs to the `Projects and Issues` category. I've gone ahead and moved it for you. Good luck!
